<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SnapToGridDialogClass</class>
 <widget class="QDialog" name="SnapToGridDialogClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>261</width>
    <height>155</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Snap To Grid Settings</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../icons/SliceViewerIcons.qrc">
    <normaloff>:/SliceViewer/icons/grid.png</normaloff>:/SliceViewer/icons/grid.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Enter the interval for snap-to-grid:
(0 for no snap in this direction)</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="lblSnapX">
       <property name="text">
        <string>Snap X:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="txtSnapX">
       <property name="toolTip">
        <string>Enter the interval to snap to in the X direction</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lblSnapY">
       <property name="text">
        <string>Snap Y:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="txtSnapY">
       <property name="toolTip">
        <string>Enter the interval to snap to in the Y direction</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnOK">
       <property name="text">
        <string>OK</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../../icons/SliceViewerIcons.qrc"/>
 </resources>
 <connections/>
</ui>
